pkg/x/vmwscript/examples/wordpress.json

```diff
@@ -0,0 +1,145 @@
+                    "cmd":"/usr/bin/firewall-cmd --permanent --zone=public --add-service=https",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Reload firewall rules",            
+                    "cmd":"/usr/bin/firewall-cmd --reload",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Install web-server/mariadb/php packages",            
+                    "cmd":"/bin/yum -y install mariadb-server httpd php php-mysql",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Enable DB on Boot and start",            
+                    "cmd":"/usr/bin/systemctl enable mariadb.service; /usr/bin/systemctl start mariadb.service",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Configure Database",            
+                    "cmd":"echo 'CREATE DATABASE wordpress;' | mysql; echo \"GRANT ALL PRIVILEGES ON wordpress.* TO 'wordpress'@'localhost' IDENTIFIED BY 'password';\" | mysql; echo \"FLUSH PRIVILEGES;\" | mysql",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Download wordpress",            
+                    "cmd":"curl -O https://wordpress.org/latest.tar.gz",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Extracting Wordpress installation",            
+                    "cmd":"tar -C /var/www/html/ --strip-components=1 -zxvf latest.tar.gz && rm -f latest.tar.gz",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Creating directories and set directory permissions",            
+                    "cmd":"mkdir -p /var/www/html/wp-content{uploads,cache}; chown apache:apache /var/www/html/wp-content{uploads,cache}",
+                    "sudoUser":"root"
+                   },
+                   {
+                    "type":"execute",                    
+                    "note":"Create Wordpress configuration file and add configuration details",            
+                    "cmd":"cp /var/www/html/wp-config-sample.php /var/www/html/wp-config.php; sed -i 's@database_name_here@wordpress@' /var/www/html/wp-config.php; sed -i 's@username_here@wordpress@' /var/www/html/wp-config.php;sed -i 's@password_here@password@' /var/www/html/wp-config.php;",
```

Comment:
Use the playbook to turn these into flags / prompts from the user when running the script?